packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.test.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ResetPasswordConfirmed from '.';
+import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
+import { FluentBundle } from '@fluent/bundle';
+import { usePageViewEvent } from '../../lib/metrics';
+
+describe('ResetPasswordConfirmed', () => {
+  let bundle: FluentBundle;
+  beforeAll(async () => {
+    // what bundle should we be getting here?
```

Comment:
This looks like the right bundle to me. Taking note that I'll need to add these tests to the ResetPassword component too.